js/app.js

```diff
@@ -1 +1,92 @@
-console.log('Hello frontend');
```

Comment:
Code is straightforward but could use commenting on each function to support understanding of the javascript code and how it effects game flow.